pwnlib/asm.py

```diff
@@ -183,6 +191,7 @@ def which_binutils(util, check_version=False):
         'i686':   ['x86_64', 'amd64'],
         'amd64':  ['x86_64', 'i386'],
         'mips64': ['mips'],
```

Comment:
So I think the issue would be solved by using 'mips', 'mipsel' here and for 'mips'